.github/workflows/ci.yml

```diff
@@ -24,9 +24,18 @@ jobs:
           path: ~/.m2
           key: ${{ runner.os }}-m2-${{ hashFiles('**/pom.xml') }}
           restore-keys: ${{ runner.os }}-m2
+      - name: Clone CCM
+        uses: actions/checkout@v2
+        with:
+          repository: dimas-b/ccm
```

Comment:
Why the special repo? Do we think this'll be removed by the time the PR is merged?

Reply:
yeah, because my CCM PR is not merged yet. I'll fix before merging.

Reply:
riptano/ccm#719 has been merged, we're using that repo in CI now.